test/example.c

```diff
@@ -805,50 +806,96 @@ void test_deflate_pending(unsigned char *compr, size_t comprLen) {
 }
 
 /* ===========================================================================
- * Test deflatePrime() with small buffers
+ * Test deflatePrime() wrapping gzip around deflate stream
  */
-void test_deflate_prime(unsigned char *compr, size_t comprLen) {
+void test_deflate_prime(unsigned char *compr, size_t comprLen, unsigned char *uncompr, size_t uncomprLen) {
```

Comment:
This is bad... Changes in function signature break build.

Reply:
Fixed thanks.